docs/EVALUATION.md

```diff
@@ -0,0 +1,372 @@
+
+Evaluation at regular intervals would look sth like this:
+
+```mermaid
+gantt
+    title Pipeline run
+    dateFormat YYYY-MM-DD
+    axisFormat %Y
+    section models
+        Trigger1 :milestone, 1930, 0d
+        model 1          :a1, 1930, 10y
+        Trigger2 :milestone, :after a1
+        model 2          :a2, after a1, 5y
+        Trigger3 :milestone, :after a2
+        model 3          :a3, after a2, 5y
+        Trigger4 :milestone, :after a3
+        model 4          :a4, after a3, 25y
+        Trigger5 :milestone, :after a4
+        model 5          :a5, after a4, 25y
+        Trigger6 :milestone, :after a5
+        model 6          :a6, after a5, 5y
+        Trigger7 :milestone, :after a6
+        model 7          :a7, after a6, 5y
+    section local evaluations (for model 4)
+        Evaluation4.1 :milestone, 1950, 0d
+        Evaluation4.1 :milestone, 1954, 0d
+        Evaluation4.1 :milestone, 1958, 0d
+        Evaluation4.1 :milestone, 1962, 0d
+        Evaluation4.1 :milestone, 1966, 0d
+        Evaluation4.1 :milestone, 1970, 0d
+    section local evaluation data (for model 4): +- 5 years
+        data4.1 :e4.1, 1947, 6y
+        data4.2 :e4.2, 1951, 6y
+        data4.3 :e4.3, 1955, 6y
+        data4.4 :e4.4, 1959, 6y
+        data4.5 :e4.5, 1963, 6y
+        data4.6 :e4.6, 1967, 6y
+```
+
+<details>
+  <summary><b>Plot 4 (matrix plot, x=time up to which samples where used for training, y=model (categorical), color=accuracy of model at this point)</b></summary>
+
+Compared to the current approach this diagram integrates a fair representation of "time" (x-axis). As triggers aren't spaced equally in an abstract trigger policy the current assumptions don't hold anymore. Also this brings the advantage that we would see how drift / model degradation accumulates over **realtime** (not trigger indexes).
+
+</details>
+
+##### 3.2.2.1 Total pipeline performance
+
+Next, we start considering not only the performance of a pipeline, but also the cost of running the pipeline - which is mainly made up of training costs, policy evaluation costs and data loading costs.
+
+Continuing from the previous section, we are left with the following dimensions:
+
+- time
+- evaluation metric value
+- pipeline run
+- cost
+
+For a cost/performance tradeoff comparison there are two useful plotting strategies:
+
+<details>
+  <summary><b>Plot 2 + Plot 3 (stacked barplot, x=time, y=cost | eval metric value (ensemble of all models), color=pipeline)</b></summary>
```

Comment:
i don't fully understand the Plot numbering here yet. Is this just "notes" in some sense? Did you plan to add screenshots here?

Reply:
I didn't plan to add screenshots. Here the numbers were just used as identifiers to refer to certain plots from different sections.

Reply:
Added some sample plots.